.github/workflows/deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: 🐍  Publish to PyPI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+jobs:
+  build-and-publish:
+    name: Build and publish Python 🐍 distributions to PyPI
+    if: startsWith(github.event.ref, 'refs/tags')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+          architecture: 'x64'
+
+      - name: Build Package and Check
+        run: |
+          python -m pip install --upgrade setuptools wheel twine
+          python setup.py sdist bdist_wheel
+          python -m twine check dist/*
+
+      - name: Deploy to PyPi
+        uses: pypa/gh-action-pypi-publish@master
```

Comment:
Please use a specific version, I checked the repo and the latest is `v1.3.1`

Reply:
right latest tag